### PR TITLE
CML-22411 Corrected the Migration Script Failure

### PR DESCRIPTION
**Pull request by navichahal3**
https://issues.sonatype.org/browse/CLM-22411

### CODE REVIEW

**Review by koraytugay — APPROVED** [review]
+1 LGTM

File: iq-config-as-code/iq-scrape-conf.py [hunk 1]
```diff
@@ -104,20 +104,22 @@ def main():
...
-                od = {}
-                od['organizations'] = []
-                od['organizations'].append(org_conf)
-                if (org_conf['name'] == 'Root Organization'):
-                    orgs.insert(0, org_conf)
-                else:
-                    orgs.append(org_conf)
-                persist_data(od, f'{output_dir}{get_organization_name(org["id"])}-config.json')
+            if applications is not None:
+                app =[]
```
Comment (koraytugay) [inline iq-config-as-code/iq-scrape-conf.py:108]:
```suggestion
                app = []
```

Comment (koraytugay) [inline iq-config-as-code/iq-scrape-conf.py:108]: Probably this is not needed, is it?

**Review by koraytugay — CHANGES_REQUESTED** [review]
Please see comments.

File: iq-config-as-code/iq-scrape-conf.py (hunk 1 above)
Comment (navichahal3) [reply iq-config-as-code/iq-scrape-conf.py]: d60935ddde7464ea5c4098e6dcfdb7f384827263

**Review by koraytugay — APPROVED** [review]
+1 LGTM

**Pull request merged**